x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/edit_warning.tsx

```diff
@@ -102,6 +103,30 @@ export const EditWarning: FunctionComponent = () => {
             <EuiSpacer />
           </>
         )}
+        {isDeprecatedPolicy && (
+          <>
+            <EuiCallOut
+              title={
+                <FormattedMessage
+                  id="xpack.indexLifecycleMgmt.editPolicyModal.deprecatedPolicyTitle"
+                  defaultMessage="This policy is deprecated"
+                />
+              }
+              color="warning"
+              iconType="warning"
+              data-test-subj="editPolicyWithDeprecation"
+            >
+              <p>
+                <FormattedMessage
+                  id="xpack.indexLifecycleMgmt.editPolicyModal.deprecatedPolicyDescription"
+                  defaultMessage="This policy is deprecated and should not be relied on."
```

Comment:
```suggestion
                  defaultMessage="This policy is no longer supported and might be removed in a future release. Instead, use one of the other policies available."
```
Not entirely sure about that 2nd sentence so feel free to adjust, but for a deprecation message I'd really like if we can be explicit about what users should do about it if they're using it.

Reply:
Thanks a lot for the suggestion, it makes total sense to me!